<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestReportGenerator</class>
 <widget class="QMainWindow" name="TestReportGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestReportGenerator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>470</width>
      <height>264</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="checkBox1">
         <property name="text">
          <string>Ch1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox2">
         <property name="text">
          <string>Ch2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox3">
         <property name="text">
          <string>Ch3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox4">
         <property name="text">
          <string>Ch4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox5">
         <property name="text">
          <string>Ch5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox6">
         <property name="text">
          <string>Ch6</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox7">
         <property name="text">
          <string>Ch7</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox8">
         <property name="text">
          <string>Ch8</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="topMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Input file (.csv)</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="inputFileFIeld"/>
         </item>
         <item>
          <widget class="QPushButton" name="BrowseCSV">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Output folder</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="OutputFileField"/>
       </item>
       <item>
        <widget class="QPushButton" name="BrowseOutput">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Approved by</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="approveBy"/>
     </item>
     <item>
      <widget class="QPushButton" name="Generate">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>472</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
